vars/rollingOperatorUpgradePipeline.groovy

```diff
@@ -22,6 +22,10 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('availability_zone', 'b')}",
                description: 'Availability zone',
                name: 'availability_zone')
+            string(defaultValue: "${pipelineParams.get('k8s_version', '')}",
```

Comment:
Both versions (base and target) should be configurable, as we may want to test upgrades between patch releases as well. Target can be optional with same behavior it's now.

Reply:
Target versioning will require more changes. It is not configurable as of now. Only implicit increment.